pkg/deploy/lattice/target_group_manager_test.go

```diff
@@ -1105,3 +1140,112 @@ func Test_Get(t *testing.T) {
 		}
 	}
 }
+
+func Test_defaultTargetGroupManager_getDefaultHealthCheckConfig(t *testing.T) {
+	var (
+		resetValue     = aws.Int64(0)
+		defaultMatcher = &vpclattice.Matcher{
+			HttpCode: aws.String("200"),
+		}
+		defaultPath     = aws.String("/")
+		defaultProtocol = aws.String(vpclattice.TargetGroupProtocolHttp)
+	)
+
+	type args struct {
+		targetGroupProtocolVersion string
+	}
+
+	tests := []struct {
+		name string
+		args args
+		want *vpclattice.HealthCheckConfig
+	}{
+		{
+			name: "HTTP1 default health check config",
+			args: args{
+				targetGroupProtocolVersion: vpclattice.TargetGroupProtocolVersionHttp1,
+			},
+			want: &vpclattice.HealthCheckConfig{
+				Enabled:                    aws.Bool(true),
+				HealthCheckIntervalSeconds: resetValue,
+				HealthCheckTimeoutSeconds:  resetValue,
+				HealthyThresholdCount:      resetValue,
+				UnhealthyThresholdCount:    resetValue,
+				Matcher:                    defaultMatcher,
+				Path:                       defaultPath,
+				Port:                       nil,
+				Protocol:                   defaultProtocol,
+				ProtocolVersion:            aws.String(vpclattice.HealthCheckProtocolVersionHttp1),
+			},
+		},
+		{
+			name: "empty target group protocol version default health check config",
+			args: args{
+				targetGroupProtocolVersion: vpclattice.TargetGroupProtocolVersionHttp1,
+			},
```

Comment:
For this unit test, "empty target group protocol version default health check config". Do you need to put empty targetGroupProtocolVersion?